Makefile.am

```diff
@@ -182,3 +182,18 @@ AM_DISTCHECK_CONFIGURE_FLAGS =		\
 	--enable-documentation		\
 	--disable-maintainer-mode	\
 	--enable-introspection
+
+.PHONY: coverage lcov-clean genlcov
+
+coverage:
+	$(AM_V_GEN) $(MAKE) $(AM_MAKEFLAGS) lcov-clean
+	$(AM_V_GEN) $(MAKE) $(AM_MAKEFLAGS) check
```

Comment:
I think this could do `make  -j $(getconf _NPROCESSORS_ONLN) check`

Reply:
I'll make that change